optuna/multi_objective/study.py

```diff
@@ -227,6 +227,24 @@ def optimize(
 
         Please refer to the documentation of :func:`optuna.study.Study.optimize`
         for further details.
+
+        Example:
+
+            .. testcode::
+
+                import optuna
+
+                def objective(trial):
+                    # Binh and Korn function.
+                    x = trial.suggest_float("x", 0, 5)
+                    y = trial.suggest_float("y", 0, 3)
+
+                    v0 = (4 * x) ** 2 + (4 * y) ** 2
```

Comment:
Sorry I understand that this function is taken from existing code but it seems to be wrong. I can merge this PR as is and fix this in a follow up as well. Whichever you prefer.
```suggestion
                    v0 = 4 * x ** 2 + 4 * y ** 2
```